iceoryx_utils/include/iceoryx_utils/posix_wrapper/timer.hpp

```diff
@@ -321,6 +321,58 @@ class Timer
 };
 
 } // namespace posix
+
+
+namespace clock
```

Comment:
This implementation is no posix abstraction, therefore I would suggest to move the `DeadlineTimer` into `cxx`.

Reply:
Yes. This class will be moved to a new file within iceoryx_utils/cxx/DeadlineTimer.hpp. The namespace will also be updated to iox::cxx::clock::DeadLineTimer. Is that fine?

Reply:
please no additional `clock` namespace. If we put everything into it's own namespace we::get::long::names::like::in::java :D

Reply:
Okay.  Got me right! I am from Java developer background :)

Reply:
Hahaha, really? I didn't know that. Well, idiomatic Java seems to be like that :D